### DISCUSSION TITLE
Issues with GitHub Student Benefits (GitHub Copilot)

### DISCUSSION THREAD

**Discussion by SebasRomaguera** [community/community#146370] [GitHub Education]
### Hi everyone,

My application was approved, but I don't have access to GitHub Copilot, and I don't know why. If anyone else is experiencing the same issue, please let me know. Thank you.

**Comment by Mayank-Garg7** [comment]
If your GitHub Copilot application is approved but you don’t have access, follow these steps to resolve the issue:

1. Check Your Subscription
Make sure you have an active GitHub Copilot subscription.
Go to [GitHub Billing Settings](https://github.com/settings/billing) and confirm Copilot is listed under your subscriptions.
If not, visit the [GitHub Copilot page](https://github.com/features/copilot/) and start a subscription.

2. Verify Your Account
Ensure you’re logged into the GitHub account where Copilot access was approved.
If you have multiple accounts, double-check you’re using the correct one.

3. Confirm Approval
Check your email for a confirmation or activation email from GitHub.
Follow any instructions provided in the email to enable Copilot.

If you find this helpful, please upvote the answer. If you still have an issue, feel free to reply here with a detailed explanation of your problem.

**Comment by davevad93** [comment]
Hi @SebasRomaguera ,

What's the status of your application?

If the label is green and states "`Congrats! Your request for @SebasRomaguera was approved and your benefits will be available soon.` that means you have to wait. It usually takes a few days, but sometimes it can take a little longer, even a couple of weeks; the waiting time may vary depending on different factors, such as the volume of requests.

If the label is purple and states "`Congrats! Your request for @SebasRomaguera was approved and your benefits are now available.`", that means you should have access to your benefits. Try first going to the [pack offers page](https://education.github.com/pack), scroll down to the "***All offers***" section and select one of the available offers, for example [Copilot](https://education.github.com/pack/redeem/copilot-student), if you can redeem it you are fine, if you can't, I suggest you to raise a ticket [here](https://support.github.com/contact/education), select "`I am a verified student having trouble redeeming my Global Campus offers`" and explain your issue.

For further reference, take a look at the [Student Developer Pack Application FAQs](https://github.com/orgs/community/discussions/111352):

- [How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)
- [How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)

And also the following discussions: #132790 , #131665 , #132470 , #133531, #134279, #135849, #136887, #138721, #139268, #140287, #144741
